pkg/dataplane/dataplane.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Cisco Systems Inc
```

Comment:
Can you update the Copyright info for the new files to match the existing copy right info.

Reply:
AFAIK the standard procedure with the apache license is to put the name of the copyright holder in the license block (which is Cisco for the files I write, even if they are contributed to a Tigera project). That does not change any of Tigera’s rights regarding this file, and you can add a Tigera line as soon as you modify them. Let me know if you’re ok with that.